.readthedocs.yaml

```diff
@@ -0,0 +1,22 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
+# Build documentation with Sphinx
+sphinx:
+  configuration: Documentation/conf.py
```

Comment:
My understanding is that it will use the "standalone HTML" builder by default, when we've been using the HTMLDir

![image](https://github.com/cilium/cilium/assets/17001771/5fe3cab4-f835-4dd7-825c-f742338a401b)

```suggestion
  configuration: Documentation/conf.py
  builder: "dirhtml"
```

About this builder:


> This is a subclass of the standard HTML builder. Its output is a directory with HTML files, where each file is called index.html and placed in a subdirectory named like its page name. For example, the document markup/rest.rst will not result in an output file markup/rest.html, but markup/rest/index.html. When generating links between pages, the index.html is omitted, so that the URL would look like markup/rest/.

(https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.html.DirectoryHTMLBuilder)